tiny-java-containers/setup-musl.sh

```diff
@@ -10,7 +10,7 @@ rm x86_64-linux-musl-native.tgz
 
 # Download, build, install zlib into TOOLCHAIN_DIR
 echo "zlib version=${ZLIB_VERSION}"
-wget -q https://zlib.net/zlib-${ZLIB_VERSION}.tar.gz
+wget -q https://zlib.net/fossils/zlib-${ZLIB_VERSION}.tar.gz
```

Comment:
Good to know this is working now. Our GitHub action has been building zlib from source for a while:
https://github.com/graalvm/setup-graalvm/blob/70e1936e9c8008d57b5227a739cf7b45750390ac/src/features.ts#L26-L45